src/main/resources/changelog.xml

```diff
@@ -32,6 +32,23 @@
             <column name="animal_id" type="long">
             </column>
         </createTable>
+        <createTable tableName="users">
```

Comment:
do you remember what I said about Liquibase changeSet? once you have a changeSet commited to master, you can never change it again. So please move this to another changeSet

This would cause your application not being able to start on production.

Reply:
I guess I understood it wrong.